packages/block-editor/src/components/off-canvas-editor/link-ui.js

```diff
@@ -137,7 +137,6 @@ export function LinkUI( props ) {
 			placement="bottom"
 			onClose={ props.onClose }
 			anchor={ props.anchor }
-			__unstableSlotName={ '__unstable-block-tools-after' }
```

Comment:
@jorgefilipecosta This was added in [this discussion](https://github.com/WordPress/gutenberg/pull/47930#issuecomment-1436647168).

If we change here do we also need to change the other `link-ui.js` file?

Reply:
I updated the PR to also change the other file. I'm not seeing regressions.

Reply:
Hopefully this ok then. Thanks 👍